src/proxy/chain.ts

```diff
@@ -27,24 +27,29 @@ const pullActionChain: ((req: any, action: Action) => Promise<Action>)[] = [
   proc.push.checkRepoInAuthorisedList,
 ];
 
+const defaultActionChain: ((req: any, action: Action) => Promise<Action>)[] = [
```

Comment:
Adding this to ensure that all requests are validated against the allowed repos, regardless of type, reducing the likelihood of the proxy being abused to forward request onto Github for arbitrary/unknown repos.